test/EFCore.PG.FunctionalTests/Query/NorthwindAggregateOperatorsQueryNpgsqlTest.cs

```diff
@@ -20,13 +20,13 @@ public override async Task Contains_with_local_uint_array_closure(bool async)
 
             // Note: PostgreSQL doesn't support uint, but value converters make this into bigint
             AssertSql(
-                @"@__ids_0='System.Int64[]' (DbType = Object)
+                @"@__ids_0='System.Int32[]' (DbType = Object)
```

Comment:
How this is related to changes?

Reply:
It isn't really, I just discovered it together while investigating why we don't already have failing tests on this. The reason is that in the tests' Northwind model, we map the price properties to decimal and not to money like SqlServer, since money is discouraged in PG...

The commits are separate though.